src/wms/src/main/java/org/geoserver/wms/capabilities/GetCapabilitiesTransformer.java

```diff
@@ -100,8 +98,13 @@
  *      org.geoserver.platform.Operation)
  */
 public class GetCapabilitiesTransformer extends TransformerBase {
-    /** fixed MIME type for the returned capabilities document */
-    public static final String WMS_CAPS_MIME = "application/vnd.ogc.wms_xml";
+    /** default MIME type for the returned capabilities document */
+    public static final String WMS_CAPS_DEFAULT_MIME = "application/vnd.ogc.wms_xml";
```

Comment:
I was under the impression the different WMS specifications have different defaults (example WMS 1.3.0 has text/xml" as a default ... 

Okay I see there is a Capabilities_1_3_0_Transformer (which does not have this format option).